swap/src/asb/network.rs

```diff
@@ -163,6 +172,12 @@ pub mod behaviour {
         }
     }
 
+    impl From<IdentifyEvent> for OutEvent {
+        fn from(_: IdentifyEvent) -> Self {
+            OutEvent::Other
```

Comment:
Maybe this needs its own `OutEvent`?

Reply:
I saw that but then wasn't sure. Will have a closer look when I can.